submissions/Vlada-khom/html-css-popup/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <title>Popup</title>
+  </head>
+  <body class="background">
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <a href="#" class="navLink">Mail</a>
+          </li>
+          <li>
+            <a href="#" class="navLink">Images</a>
+          </li>
+          <li class="burger">
+            <input type="checkbox" id="burgerCheckbox" class="input" />
+            <label for="burgerCheckbox" class="burgerLabel navLink">
+              <img
+                class="imgLink burgerImg"
+                src="./img/popup-button.png"
+                alt="burger menu"
+              />
+            </label>
+            <div class="popupContainer main">
+              <div class="visibleBlock containerForItem">
+                <div class="itemBlock">
+                  <a href="#" class="itemPopup">
+                    <div class="imgBlock">
+                      <img src="./img/calendar.png" alt="calendar" />
+                    </div>
+                    Calendar
+                  </a>
+                </div>
```

Comment:
Why you use `ul>li>a` for navigation on the top level, but inside the popup, you use `div`?

Reply:
Initially I use div but I didn't set up switch popup through tab and I changed div on li and add <a>. Because all work and other code has been written I decided didn't change it.I know that tab works because I use <a> but I changed css under li and didn't want to change it.

Reply:
For nested navigation better to use lists, it helps screen-readers and search bots to understand the structure of your menu)
In your case replace `div.itemBlock` with `li.itemBlock` and add `ul` as wrapper)